### DISCUSSION TITLE
What is different from "field $FOO : common" and "our $FOO".

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [Ovid/Cor#71] [Q&A] [answered]
What is different from "field $FOO : common" and "our $FOO". 

https://github.com/Ovid/Cor/blob/master/rfc/attributes.md#6311-common

```
class MyClass {
  our $FOO;
}

class MyClass {
  field $FOO : common;
}
```

And Is "our $FOO" allowed in the class syntax?

**Comment by haarg** [comment, marked as answer]
An `our` variable is publicly visible. `our $FOO` can be accessed via `$MyClass::FOO`, but `field $FOO : common` can't.

In a class context, `field $FOO : common;` is virtually identical to `my $FOO;`.

**Reply by yuki-kimoto** [reply]
Thanks! I understood now.
